packages/common/src/chains/goerli.json

```diff
@@ -26,7 +26,7 @@
     },
     {
       "name": "dao",
-      "block": 0,
+      "block": null,
       "forkHash": "0xa3f5ab08"
```

Comment:
So I do get this right that DAO support is not active neither on Goerli nor on Kovan, correct?

Then this change makes a lot of sense and is technically also a bug fix along the way. Wonder if we should generally remove all the `null` `DAO` entries from `Common` completely since these are just HFs not being present on the respective chains (and never will be).

For `null` I always had the semantics "HF didn't happen yet but has some settings and will happen some time in the future" in mind, so e.g. the state of the `Berlin` HF right now. Technically it shouldn't make any difference though.

Reply:
I do get the confusion here. But just for comparision:

`geth`: `config="{ChainID: 1 Homestead: 1150000 DAO: 1920000 DAOSupport: true (...)`

`geth --goerli`: `config="{ChainID: 5 Homestead: 0 DAO: <nil> DAOSupport: true (...)`

I just checked all 4 testnets and neither of them has any contract/code at "bf4ed7b27f1d666546e30d74d50d173d20bca754" (the DAO withdraw contract on mainnet)

This is thus also reflected in all 4 testnet common files now (they all have the DAO block set to `null`)